internal/config/config.go

```diff
@@ -279,3 +381,13 @@ func (tc *TestsConfig) toMap() (map[string]status, error) {
 
 	return res, nil
 }
+
+// CheckDuplicates checks for duplicate test names or prefixes within a *TestConfig instance.
+// It returns an error if any duplicates are found.
+func (tc *TestConfig) CheckDuplicates() error {
```

Comment:
That should be in the `configload` package. Once the config is loaded, it is valid, does not contain duplicates, etc